Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM rethinkdb:2.3
 
-RUN apt-get update
+# From http://stackoverflow.com/a/38553499
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
```

Comment:
Each command in the Dockerfile generates a new image.  I grouped those commands together as one so that a single image is produced if the command succeed.  This follows the general pattern found at https://docs.docker.com/engine/reference/builder/